server/datastore/mysql/hosts.go

```diff
@@ -574,6 +574,7 @@ SELECT
   COALESCE(hst.seen_time, h.created_at) AS seen_time,
   t.name AS team_name,
   COALESCE(hu.software_updated_at, h.created_at) AS software_updated_at,
+  DATE_SUB(COALESCE(h.detail_updated_at, h.created_at), INTERVAL uptime/1000 MICROSECOND) as last_restarted,
```

Comment:
If `detail_updated_at` is `NULL` it means the host never checked in with `uptime` data. So there should be an `undefined` (or `NULL`?) value for `last_restarted`.